bin/epaa.py

```diff
@@ -531,8 +531,8 @@ def create_metadata_column_value(pep, c):
     transcript_ids = [x.transcript_id for x in set(pep[0].get_all_transcripts())]
     variants = []
     for t in transcript_ids:
-        variants.extend([v for v in pep[0].get_variants_by_protein(t)])
-    meta = set([str(y.get_metadata(c)[0]) for y in set(variants)])
+        variants.extend([v for v in pep[0].get_variants_by_protein(t)]) 
```

Comment:
Don't know if it will raise errors, but maybe remove the space that you added on line 534

Reply:
This should be removed otherwise one of the checks won't pass (trailing whitespaces).